CMakeLists.txt

```diff
@@ -21,13 +21,24 @@
 #
 # ##############################################################################
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.10)
+
+# ROCM Path
+if(DEFINED ENV{ROCM_PATH})
+  set(ROCM_PATH $ENV{ROCM_PATH} CACHE PATH "Default ROCm installation path")
+elseif(ROCM_PATH)
+  message("-- INFO:ROCM_PATH Set -- ${ROCM_PATH}")
+else()
+  set(ROCM_PATH /opt/rocm CACHE PATH "Default ROCm installation path")
+endif()
+
+# Set AMD Clang as default compiler
 if (NOT DEFINED CMAKE_CXX_COMPILER)
-    set(CMAKE_CXX_COMPILER clang++)
+  set(CMAKE_CXX_COMPILER ${ROCM_PATH}/bin/amdclang++)
```

Comment:
please change `${ROCM_PATH}/bin/amdclang++`  to `${ROCM_PATH}/lib/llvm/bin/clang++`